openassessment/xblock/staff_area_mixin.py

```diff
@@ -510,10 +511,15 @@ def clear_team_state(self, user_id, course_id, item_id, requesting_user_id, subm
             "Student and team state cleared",
             requesting_user_id
         )
+
         # Tell the submissions API to orphan the submissions to prevent them from being accessed
         from submissions import team_api as team_submissions_api
         team_submissions_api.reset_scores(team_submission_uuid)
 
+        # Clean up shared files for the team
+        team_id = team_submissions_api.get_team_submission(team_submission_uuid).get('team_id', None)
+        delete_shared_files_for_team(course_id, item_id, team_id)
```

Comment:
How does `delete_shared_files_for_team` behave when the `team_id` is `None` ? Does it make sense to add a test case for this?

Reply:
It can't have a `team_id` of None, since a `TeamSubmission` must have a `team_id`.